ControllerKey.js

```diff
@@ -116,18 +118,19 @@ export class ControllerKey {
    * wrap or unwrap keys.
    *
    * If this ControllerKey is a controller of the KEK, then the API for it can
-   * returned by passing only the key description. Otherwise, an OCAP-LD
-   * authorization capability must also be passed; without this capability,
-   * calls to the returned API will not be authorized to perform KEK operations.
+   * be returned by passing only the key id and type. Otherwise,
+   * an OCAP-LD authorization capability must also be passed;
+   * without this capability, calls to the returned API will not be authorized
+   * to perform KEK operations.
```

Comment:
The above is still very sub-optimally spaced :( ... doesn't have to be perfect, but looks pretty goofy right now.

Reply:
Addressed in https://github.com/digitalbazaar/web-kms-client/pull/2/commits/09d4ec93574132f2a7a5205559256f63a81ca903